core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.spec.ts

```diff
@@ -131,6 +135,30 @@ describe('Contributor dashboard page', function() {
       expect(ctrl.activeTabName).toBe(changedTab);
       expect(ctrl.showLanguageSelector()).toBe(true);
     });
+
+    it('should call scrollFunction on scroll', function() {
+      var e = document.createEvent('Event');
+      var scrollSpy = spyOn(ctrl, 'scrollFunction');
+      e.initEvent('scroll', true, true);
+      window.dispatchEvent(e);
+      expect(scrollSpy).toHaveBeenCalled();
+    });
+
+    it('should not be viewed the collapsed header content if' +
+      ' scrolled less than 5', function() {
```

Comment:
Remember the full test name will depend on the nested structure; in this case it would be: "Contributor dashboard page when used is logged in should not be viewed the collapsed header content if scrolled less than 5".  So "should not be viewed" doesn't really make sense here as the contributor dashboard page in the abstract is the subject of the sentence. Can we update the test name to be something like "should show default header if window scrollTop is less than 5"? Ditto below.

Reply:
Done